dbschema/language.esdl

```diff
@@ -72,11 +72,13 @@ module default {
       or not exists .firstScriptureEngagement
     );
     
-    multi link engagements := (
+    engagements := (
       # Similar to previous version but avoids https://github.com/edgedb/edgedb/issues/5846
       select LanguageEngagement filter __source__ = .language
     );
-    
+    projects := (
+      select TranslationProject filter __source__ = .languages
+    );
```

Comment:
Doing it this way as it keeps the `TranslationProject` sub-type.
Where as `.projectContext.projects` is just `Project`, and
`.engagements.project` is incorrectly just `Project`.